handler/src/main/java/io/netty/handler/ssl/OpenSslClientContext.java

```diff
@@ -67,7 +85,25 @@ public OpenSslClientContext(File certChainFile) throws SSLException {
      */
     @Deprecated
     public OpenSslClientContext(TrustManagerFactory trustManagerFactory) throws SSLException {
-        this(null, trustManagerFactory);
+        this(null, trustManagerFactory, KeyStore.getDefaultType());
+    }
+
+    /**
+     * Creates a new instance.
+     *
+     * @param certChainFile an X.509 certificate chain file in PEM format.
+     *                      {@code null} to use the system default
+     * @param trustManagerFactory the {@link TrustManagerFactory} that provides the {@link TrustManager}s
+     *                            that verifies the certificates sent from servers.
+     *                            {@code null} to use the default.
+     * @param keyStore the keystore type that should be used
+     * @deprecated use {@link SslContextBuilder}
+     */
+    @Deprecated
+    public OpenSslClientContext(File certChainFile,
```

Comment:
It does not make sense to expose a protected method and mark it as @Deprecated from the beginning. Maybe you should remove the annotation and make it package-private ?